portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,19 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
+      <h1>Hi, my name is Rashad.</h1>
       <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <p>Click here to started</p>
```

Comment:
Did you mean "to start" or "to get started"?

Reply:
neither, that was supposed to be removed for a feature I was going to add. Good catch